scripts/evaluation_script.py

```diff
@@ -48,3 +53,55 @@
     pic_loc="output/",
     pic_name="weekly_predicted_uptake.png",
 )
+
```

Comment:
I think it would be really useful to add this evaluation and plotting script to the end of main.py in the /scripts folder, rather than writing a separate script. main.py already produces an LIUM with both incident and cumulative projection attributes, plus `split_train_test` can be used to make your test data. So calculating and plotting evaluation metrics could be a natural extension of that script. More importantly, it would force you to use the output of my code as the input to yours, so that we can discover any issues at this interface.

Reply:
Or, as Scott suggested, we'll use a makefile to essentially execute our scripts sequentially. Even in this case, however, `get_uptake_data` doesn't exist as a function anymore. It has been superseded by `parse_nis` which can be given its arguments from a .yaml config.